tests/from.bats

```diff
@@ -12,12 +12,10 @@ load helpers
 
   cid=$(buildah from --pull --signature-policy ${TESTSDIR}/policy.json dir:${elsewhere})
   buildah rm $cid
-  buildah rmi ${elsewhere}
   [ "$cid" = elsewhere-img-working-container ]
 
   cid=$(buildah from --pull-always --signature-policy ${TESTSDIR}/policy.json dir:${elsewhere})
   buildah rm $cid
-  buildah rmi ${elsewhere}
```

Comment:
Ditto

Reply:
Actually removing the  ${elsewhere} image wasn't working based on my testing.   I think it was failing quietly.

```
# buildah rmi ${elsewhere}
could not get image "/tmp/image1": identifier is not an image
[root@localhost buildah]# buildah rmi dir:${elsewhere}
could not get image "dir:/tmp/image1": error reading image using reference "dir:/tmp/image1": image not known
# buildah images
IMAGE ID             IMAGE NAME                                               CREATED AT             SIZE
5b0d59026729         docker.io/library/busybox:latest                         Jan 24, 2018 04:29     1.299 MB
# buildah rmi 5b0d59026729
```

Reply:
Ok, here's my full test.  The busybox image is getting stored in /var/lib/containers/storeage/overlay-images.  The commit is placing a copy of the image into /tmp/image1 and that is not being removed even with 'buildah rmi --all'.  I think that's OK, but am not 100% sure.

```
# export elsewhere=/tmp/image1
# mkdir -p ${elsewhere}
# tree ${elsewhere}
/tmp/image1

0 directories, 0 files
# tree /var/lib/containers/storage/overlay-images
/var/lib/containers/storage/overlay-images
├── images.json
└── images.lock

0 directories, 2 files
# buildah from --name cnt1 busybox
Getting image source signatures
Copying blob sha256:57310166fe88e0dc63a80ca5c219283a932db0f3969712e2f8a86ada143bf566
 706.12 KiB / 706.12 KiB [==================================================] 0s
Copying config sha256:5b0d59026729b68570d99bc4f3f7c31a2e4f2a5736435641565d93e7c25bd2c3
 1.46 KiB / 1.46 KiB [======================================================] 0s
Writing manifest to image destination
Storing signatures
cnt1
# tree ${elsewhere}
/tmp/image1

0 directories, 0 files
# tree /var/lib/containers/storage/overlay-images
/var/lib/containers/storage/overlay-images
├── 5b0d59026729b68570d99bc4f3f7c31a2e4f2a5736435641565d93e7c25bd2c3
│   ├── =c2hhMjU2OjViMGQ1OTAyNjcyOWI2ODU3MGQ5OWJjNGYzZjdjMzFhMmU0ZjJhNTczNjQzNTY0MTU2NWQ5M2U3YzI1YmQyYzM=
│   └── manifest
├── images.json
└── images.lock

1 directory, 4 files

# buildah commit cnt1 dir:${elsewhere}
Getting image source signatures
Copying blob sha256:57310166fe88e0dc63a80ca5c219283a932db0f3969712e2f8a86ada143bf566
 706.12 KiB / 706.12 KiB [==================================================] 0s
Copying blob sha256:4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1
 32 B / 32 B [==============================================================] 0s
Copying config sha256:0730070e2a153d8e76a5aba3512fce65b2f1fdf176afc3b98281e13b612d45f7
 711 B / 711 B [============================================================] 0s
Writing manifest to image destination
Storing signatures

# tree ${elsewhere}
/tmp/image1
├── 0730070e2a153d8e76a5aba3512fce65b2f1fdf176afc3b98281e13b612d45f7.tar
├── 4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1.tar
├── 57310166fe88e0dc63a80ca5c219283a932db0f3969712e2f8a86ada143bf566.tar
├── manifest.json
└── version

0 directories, 5 files
# tree /var/lib/containers/storage/overlay-images
/var/lib/containers/storage/overlay-images
├── 5b0d59026729b68570d99bc4f3f7c31a2e4f2a5736435641565d93e7c25bd2c3
│   ├── =c2hhMjU2OjViMGQ1OTAyNjcyOWI2ODU3MGQ5OWJjNGYzZjdjMzFhMmU0ZjJhNTczNjQzNTY0MTU2NWQ5M2U3YzI1YmQyYzM=
│   └── manifest
├── images.json
└── images.lock

1 directory, 4 files

# buildah images
IMAGE ID             IMAGE NAME                                               CREATED AT             SIZE
5b0d59026729         docker.io/library/busybox:latest                         Jan 24, 2018 04:29     1.299 MB
# buildah containers
CONTAINER ID  BUILDER  IMAGE ID     IMAGE NAME                       CONTAINER NAME
f0588e8de0ff     *     5b0d59026729 docker.io/library/busybox:latest cnt1

# buildah rm cnt1
f0588e8de0ff7276ce4e0827740069e86c95413d8143b1a0b4c44b153c82f30a

# tree ${elsewhere}
/tmp/image1
├── 0730070e2a153d8e76a5aba3512fce65b2f1fdf176afc3b98281e13b612d45f7.tar
├── 4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1.tar
├── 57310166fe88e0dc63a80ca5c219283a932db0f3969712e2f8a86ada143bf566.tar
├── manifest.json
└── version

0 directories, 5 files
# tree /var/lib/containers/storage/overlay-images
/var/lib/containers/storage/overlay-images
├── 5b0d59026729b68570d99bc4f3f7c31a2e4f2a5736435641565d93e7c25bd2c3
│   ├── =c2hhMjU2OjViMGQ1OTAyNjcyOWI2ODU3MGQ5OWJjNGYzZjdjMzFhMmU0ZjJhNTczNjQzNTY0MTU2NWQ5M2U3YzI1YmQyYzM=
│   └── manifest
├── images.json
└── images.lock

1 directory, 4 files

# buildah rmi ${elsewhere}
could not get image "/tmp/image1": identifier is not an image
# buildah rmi dir:${elsewhere}
could not get image "dir:/tmp/image1": error reading image using reference "dir:/tmp/image1": image not known
# buildah images
IMAGE ID             IMAGE NAME                                               CREATED AT             SIZE
5b0d59026729         docker.io/library/busybox:latest                         Jan 24, 2018 04:29     1.299 MB
# buildah rmi 5b0d59026729
5b0d59026729b68570d99bc4f3f7c31a2e4f2a5736435641565d93e7c25bd2c3

# tree ${elsewhere}
/tmp/image1
├── 0730070e2a153d8e76a5aba3512fce65b2f1fdf176afc3b98281e13b612d45f7.tar
├── 4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1.tar
├── 57310166fe88e0dc63a80ca5c219283a932db0f3969712e2f8a86ada143bf566.tar
├── manifest.json
└── version

0 directories, 5 files
# tree /var/lib/containers/storage/overlay-images
/var/lib/containers/storage/overlay-images
├── images.json
└── images.lock

0 directories, 2 files

# buildah rmi --all
# buildah rm --all
# tree ${elsewhere}
/tmp/image1
├── 0730070e2a153d8e76a5aba3512fce65b2f1fdf176afc3b98281e13b612d45f7.tar
├── 4f4fb700ef54461cfa02571ae0db9a0dc1e0cdb5577484a6d75e68dc38e8acc1.tar
├── 57310166fe88e0dc63a80ca5c219283a932db0f3969712e2f8a86ada143bf566.tar
├── manifest.json
└── version
```